src/store/reducers/dform/initial/styles.json

```diff
@@ -545,8 +545,8 @@
                   "label": "PND"
                 },
                 {
-                  "id": "easy_fatigability",
-                  "label": "Easy Fatigability"
+                  "id": "fatigue",
```

Comment:
All `id` fields of this siblings can be modified (`sob`, `diarrhoea`). Can you please do it?

Reply:
Sure thing!